### PR TITLE
argo_stage and argo_dev: update mm setting in solrconfig

### PR DESCRIPTION
**Pull request by ndushay**
All commits are argo related;  3 are completely benign:

- add or update admin-extra.html for each argo index to document its purpose
- update schema.xml name for each argo index to differentiate them (displays in Solr admin GUI)
- remove voluminous comments from old argo2 production solrconfig.xml to make it easier to read (for comparison to argo3 for troubleshooting)

and the real purpose of this PR, which is to change the mm setting for argo_stage and argo_dev 

@sul-dlss/devops 

connects to sul-dlss/argo#685   (adjust mm setting for better search results)

### CODE REVIEW

File: argo/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+this is the legacy argo2 Solr index.  When argo2 was shut down in favor of argo3, this index became stale.
```
Comment (cbeer) [inline argo/README.md]: I think if we call this file `admin-extra.html`, it'll show up in the solr admin too, which might be nice for people poking around the solr UI.

Comment (ndushay) [inline argo/README.md]: i will change them all.

**Comment by ndushay** [comment]
all changed to admin-extra.html.  :-)   Ready for merging and deploying on this end.

**Comment by ndushay** [comment]
@sul-dlss/devops    still hoping for this plus #52  to be deployed soon

**Pull request merged**